src/compiler/docs/generate-doc-data.ts

```diff
@@ -199,7 +199,10 @@ const getDocsMethods = (methods: d.ComponentCompilerMethod[]): d.JsonDocsMethod[
       name: member.name,
       returns: {
         type: member.complexType.return,
-        docs: member.docs.tags.filter((t) => t.name === 'return').join('\n'),
+        docs: member.docs.tags
+          .filter((t) => t.name === 'returns')
```

Comment:
I think we could make this a non-breaking (or at least backwards compatible) change by accepting either `@return` or `@returns` here.  Looking at [use JSDoc](https://jsdoc.app/tags-returns.html) site, the former is a synonym for the latter. FWIW, in a previous job my team and I used `@return` exclusively - I think supporting both would be OK.

This does open us up to situations where if we have _both_ `@return` and `@returns` or even duplicates:
```tsx
  /**
   * This is a description
   * @returns Nothing!
   * @return Nada!
   * @returns Not a thing!!!!!!
   */
  @Method()
  myTest() {

  }
```
would get generated as such:
```md
## Methods

### `myTest() => Promise<void>`

This is a description

#### Returns

Type: `Promise<void>`

Nothing!
Nada!
Not a thing!!!!!!
```
which is fine (IMO).

So, TL;DR, I propose we make this:
```suggestion
          .filter((t) => t.name === 'return' || t.name === 'returns')
```

Reply:
I didn't know `@return` was also allowed and see no reason not to allow it here, good catch. But note that it **is** technically a breaking change because before the output of `@return` was always `[object Object]`.

Reply:
Happy to help. And I agree that this can go in a minor release because it's very unlikely to actually break something since it's obviously a bug and therefore shouldn't be relied upon.